### PR TITLE
docs: schema update presence matrix

### PR DESCRIPTION
**Pull request by pablo-gar**
In the schema it changes the specification for the Dataset Presence Matrix. And bumps schema version to 0.2.0.

**From**  `census_obj[“census_data”][organism].ms[“RNA”].varp[“dataset_presence_matrix”]` 
**To**  `census_obj[“census_data”][organism].ms[“RNA”][“feature_dataset_presence_matrix”]`

Please note the key is "feature_dataset_presence_matrix" the word feature was added to indicate this relates to features.

### CODE REVIEW

File: docs/cell_census_schema_0.2.0.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # CELLxGENE Cell Census Schema 
 
-**Version**: 0.0.1.
+**Version**: 0.2.0.
```
Comment (atolopko-czi) [inline docs/cell_census_schema_0.2.0.md:3]: why not just `0.1.0`?

File: docs/cell_census_schema_0.2.0.md [hunk 2]
```diff
@@ -667,12 +667,14 @@ For each organism the `SOMAExperiment` MUST contain the following:
...
 	* Feature metadata – `census_obj[“census_data”][organism].ms[“RNA”].var` – `SOMAIndexedDataFrame`
-	* Feature dataset presence matrix – `census_obj[“census_data”][organism].ms[“RNA”].varp[“dataset_presence_matrix”]` – `SOMASparseNDArray`
+	* Feature dataset presence matrix – `census_obj[“census_data”][organism].ms[“RNA”][“feature_dataset_presence_matrix”]` – `SOMASparseNDArray`
 
 
 For each organism the `SOMAExperiment` SHOULD NOT contain the following:
 
 * Elements of `census_obj[“census_data”][organism].ms[“RNA”]`: 
+	* `varm`
+	* `varp`
```
Comment (atolopko-czi) [inline docs/cell_census_schema_0.2.0.md:677]: do we want to note as being empty?

File: docs/cell_census_schema_0.2.0.md [hunk 3]
```diff
@@ -718,7 +720,7 @@ The following columns MUST be included:
 </tbody>
 </table>
 
-#### Feature dataset presence matrix – `census_obj[“census_data”][organism].ms[“RNA”].varp[“dataset_presence”]` – `SOMASparseNDArray`
+#### Feature dataset presence matrix – `census_obj[“census_data”][organism].ms[“RNA”][“feature_dataset_presence_matrix”]` – `SOMASparseNDArray`
```
Comment (atolopko-czi) [inline docs/cell_census_schema_0.2.0.md:723]: It's _possible_ that someone cuts and pastes this to use in actual Python code. Of course _I_ never did that, but I happen to know that the fancy quotes need to be edited after pasting. :). Could just use ASCII double-quote here.

File: docs/cell_census_schema_0.2.0.md (hunk 2 above)
Comment (bkmartinjr) [reply docs/cell_census_schema_0.2.0.md:677]: Further - these are optional, and IMHO we should say nothing about them unless we actually use them....

Or put another way, what exactly does the above imply? That the schema _requires_ that we have an empty collection with those names?  That seems not useful...

**Review by bkmartinjr — APPROVED** [review]
thanks @pablo-gar!  Agree with Andrew's comments, but otherwise it LGTM.

**Comment by bkmartinjr** [comment]
ps. will need to mark this draft PR as ready to review at some point

File: docs/cell_census_schema_0.2.0.md (hunk 3 above)
Comment (pablo-gar) [reply docs/cell_census_schema_0.2.0.md]: I changed all quotes to ASCII

File: docs/cell_census_schema_0.2.0.md (hunk 2 above)
Comment (pablo-gar) [reply docs/cell_census_schema_0.2.0.md]: I agree, I have deleted that section now

File: docs/cell_census_schema_0.2.0.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # CELLxGENE Cell Census Schema 
 
-**Version**: 0.0.1.
+**Version**: 0.1.0.
```
Comment (atolopko-czi) [inline docs/cell_census_schema_0.2.0.md:3]: will have to rename file too now.

File: docs/cell_census_schema_0.2.0.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
...
 
 **Last edited**: Dec, 2022.
```
Comment (atolopko-czi) [inline docs/cell_census_schema_0.2.0.md:5]: update

**Review by atolopko-czi — CHANGES_REQUESTED** [review]
Versioning can be a pain--a couple more fixes needed.

**Review by atolopko-czi — APPROVED** [review]
👏 LGTM!

**Pull request merged**